CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # CHANGELOG
```

Comment:
Were these intended?

Reply:
I only added the following line.
`- implement restApi testing using pytest`

Reply:
nope

Reply:
Ah ok all good. the extra newlines won't hurt anything.